src/main/java/io/github/redouane59/twitter/dto/tweet/TweetV1.java

```diff
@@ -100,6 +109,11 @@ public String getSource() {
     return null;
   }
 
+  @Override
+  public Entities getEntities() {
```

Comment:
Just one doubt, can't it be managed natively thanks to Lombok thanks to `@Getter` annotation ?

Reply:
Yup.